src/test/java/com/puppycrawl/tools/checkstyle/checks/header/HeaderCheckTest.java

```diff
@@ -48,6 +48,36 @@ protected String getPackageLocation() {
         return "com/puppycrawl/tools/checkstyle/checks/header/header";
     }
 
+    @Test
+    public void testPackageInfoCheck() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(HeaderCheck.class);
+        checkConfig.addAttribute("skipFiles", "package-info.java");
+        checkConfig.addAttribute("header", "example");
+        createChecker(checkConfig);
+        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
+        verify(checkConfig, getPath("package-info.java"), expected);
+    }
+
+    @Test
+    public void testPackageInfoCheckFalse() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(HeaderCheck.class);
+        createChecker(checkConfig);
+        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
+        verify(checkConfig, getPath("package-info.java"), expected);
+    }
+
+    @Test
+    public void testPackageInfoCheckOtherFile() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(HeaderCheck.class);
+        checkConfig.addAttribute("headerFile", getPath("InputHeaderjava.header"));
+        checkConfig.addAttribute("ignoreLines", "");
```

Comment:
definition of this property is not required, please remove it.

Please make a test with `package-info.java` in different folders, and test do suppression only in one of them